CMakeLists.txt

```
@@ -144,10 +144,8 @@ ENDIF(USE_QPOASES)
 
 INSTALL(TARGETS ${PROJECT_NAME} EXPORT ${TARGETS_EXPORT_NAME} DESTINATION lib)
 
-IF(BUILD_TESTING)
-  SET(BOOST_COMPONENT math unit_test_framework)
-  SEARCH_FOR_BOOST()
-  ADD_SUBDIRECTORY(tests)
-ENDIF()
+SET(BOOST_COMPONENT math unit_test_framework)
+SEARCH_FOR_BOOST()
```

```suggestion
SEARCH_FOR_BOOST()
find_package(Boost REQUIRED COMPONENTS math unit_test_framework)
```
The dependency will not be exported then.